src/electionguard/constants.py

```diff
@@ -0,0 +1,78 @@
+    119359756198641231858139651428439585561105914902686985078252796680474637856752833978884422594516170665312423393830118608408063594508087813277769835084746883589963798527237870817233369094387978405585759195339509768803496494994109693743279157584139079471178850751266233150727771094796709619646350222242437970473900636242584673413224137139139346254912172628651028694427789523683070264102332413084663100402635889283790741342401259356660761075766365672754329863241692760862540151023800163269173550320623249398630247531924855997863109776955214403044727497968354022277828136634059011708099779241302941071701051050378539485717425482151777277387633806111112178267035315726401285294598397677116389893642725498831127977915200359151833767358091365292230363248410124916825814514852703770457024102738694375502049388804979035628232209959549199366986471874840784466132903083308458356458177839111623113116525230200791649979270165318729763550486200224695556789081331596212761936863634467236301450039399776963661755684863012396788149479256016157814129329192490798309248914535389650594573156725696657302152874510063002532052622638033113978672254680147128450265983503193865576932419282003012093526302631221491418211528781074474515924597472841036553107847,
+)
+
+# TEST ONLY
+MINISCULE_TEST_CONSTANTS = create_constants(157, 13, 12, 16)
+TINY_TEST_CONSTANTS = create_constants(503, 251, 2, 5)
+SMALL_TEST_CONSTANTS = create_constants(65267, 32633, 2, 3)
+MEDIUM_TEST_CONSTANTS = create_constants(
+    18446744073704586917, 65521, 281539415968996, 15463152587872997502
+)
+
+
+class PrimeOption(Enum):
+    """Option for primes to determine election constants."""
+
+    Standard = "Standard"
+    TestOnly = "TestOnly"
+
+
+def get_constants() -> ElectionConstants:
+    """Get constants for the election by the option for the primes."""
+    env_option = getenv("PRIME_OPTION")
+    option: PrimeOption = (
+        PrimeOption(env_option) if env_option is not None else PrimeOption.Standard
+    )
+
+    option_map = {
+        PrimeOption.Standard: STANDARD_CONSTANTS,
+        PrimeOption.TestOnly: MEDIUM_TEST_CONSTANTS,
+    }
+    return option_map.get(option) or STANDARD_CONSTANTS
+
+
+get_large_prime = lambda: get_constants().large_prime
```

Comment:
These functions were added as opposed to constants to ensure the testing suite can stay flexible. The environment variables can be changed quickly on the tests allowing for different testing cases which has a lot of potential weight long term. 

Before Changes
![Screen Shot 2021-07-14 at 3 17 42 PM](https://user-images.githubusercontent.com/10125297/126008203-2767287e-8887-4e7b-98ae-d6e552ff7400.png)

After Changes
![After changes](https://user-images.githubusercontent.com/10125297/126008211-f85df830-3ade-465d-afb0-4439b1c42eb7.png)

Reply:
Good to see this. I would have been nervous about going from constants to functions.